src/font.c

```diff
@@ -1074,9 +1074,14 @@ GdipPrivateAddMemoryFont(GpFontCollection *fontCollection, GDIPCONST void *memor
 	if (!memory)
 		return InvalidParameter;
 
+#ifdef WIN32
+	f = tmpfile();
+#else
 	strcpy((char *) fontfile, "/tmp/ffXXXXXX");
 
 	f = mkstemp((char*)fontfile);
+#endif
```

Comment:
`tmpfile()` is not the correct replacement since it'll try to create the file at the system root which won't work if the user doesn't have permissions. Additionally, `fontfile` is not set but we're using it in FcConfigAppFontAddFile below.

It's better to use `GetTempFileName()` + `GetTempPath()`.